sky/skylet/job_lib.py

```diff
@@ -115,7 +117,7 @@ def __lt__(self, other):
     # reserved resources and actually started running: it will set the
     # status in the DB to RUNNING.
     'PENDING': JobStatus.INIT,
-    'RUNNING': JobStatus.PENDING,
+    'RUNNING': JobStatus.SETUP,
```

Comment:
Our generated ray program will set the status of the job to PENDING if there is no `setup` specified in the task as soon as it is started (ray's status becomes RUNNING), i.e. it will be very rare that the job will be set to `SETTING_UP` by the `update_job_status`.
The case will only happens when:
1. The generated ray program starts and changed ray's status to `RUNNING`.
2. Before the generated ray program reach the line where set the job status to `PENDING`.
3. At the same time, the skylet updates the job status to `SETTING_UP`.
4. Before the generated ray program change the status to `PENDING`, the user execute `sky queue`.

Also, I think it may be fine even if the user see the status `SETTING_UP` for a job without `setup` section, since the `SETTING_UP` can also be considered as setting up some sky related runtime for that job.

Reply:
Can we add this explanation to L120 here? Just so in the future we know why and possibly how to change this behavior.